doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -304,3 +354,4 @@ flagged for backporting, in which case the status should be changed to
 .. _Integration Testing: ../tests-integration-tests
 .. _Mailing List: ../essentials/#mailing-list
 .. _Submitting Patches: ../essentials/#submitting-patches
+.. _Merging: ../merging
```

Comment:
i'd suggest use labels instead of paths for referencing a section.  see https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref and see https://github.com/ceph/ceph/pull/36879/commits/ca1cca0abc05d7e3ad4fe25adceb6f83615f66b1 for some examples.

Reply:
I think paths are fine for the urls within the same directory.

Reply:
if one moves the file around, the "path" is invalidated. please see the broken links i've been fixed recently.